Sources/SwiftDocC/Infrastructure/Symbol Graph/AccessControl+Comparable.swift

```diff
@@ -10,31 +10,14 @@
 
 import SymbolKit
 
-extension SymbolGraph.Symbol.AccessControl: Comparable {
-    private var level: Int? {
-        switch self {
-        case .private:
-            return 0
-        case .filePrivate:
-            return 1
-        case .internal:
-            return 2
-        case .public:
-            return 3
-        case .open:
-            return 4
-        default:
-            assertionFailure("Unknown AccessControl case was used in comparison.")
-            return nil
-        }
+extension AccessControlLevel {
+    init(from symbolKitAccessControlLevel: SymbolGraph.Symbol.AccessControl) {
+        self.init(symbolKitAccessControlLevel.rawValue)
     }
-    
+}
+
+extension SymbolGraph.Symbol.AccessControl: Comparable {
     public static func < (lhs: SymbolGraph.Symbol.AccessControl, rhs: SymbolGraph.Symbol.AccessControl) -> Bool {
-        guard let lhs = lhs.level,
-              let rhs = rhs.level else {
-            return false
-        }
-        
-        return lhs < rhs
+        return AccessControlLevel(from: lhs) < AccessControlLevel(from: rhs)
```

Comment:
Oh interesting! I was a little concerned about introducing the idea of comparable access levels to Swift-DocC in this PR but since it's already here and used I like how you've made it more generic and accessible. We should look at adding [`package`](https://github.com/apple/swift-evolution/blob/main/proposals/0386-package-access-modifier.md) here if/when it's approved.

Reply:
Yeah, I decided to add the type so that we wouldn't have to traffic strings or reexport SymbolKit. It would be good to have a set of vocabulary types internal to DocC